test/unit/connect-inject-deployment.bats

```diff
@@ -79,6 +79,47 @@ load _helpers
   [[ "$output" =~ "client.grpc must be true for connect injection" ]]
 }
 
+@test "connectInject/Deployment: health checks disabled by default" {
+  cd `chart_dir`
+  local cmd=$(helm template \
+      -s templates/connect-inject-deployment.yaml \
+      --set 'connectInject.enabled=true' \
+      . | tee /dev/stderr |
+      yq '.spec.template.spec.containers[0].command' | tee /dev/stderr)
+
+  local actual=$(echo "$cmd" |
+    yq 'any(contains("-enable-health-checks-controller=false"))' | tee /dev/stderr)
+  [ "${actual}" = "true" ]
+}
+
+@test "connectInject/Deployment: health checks sync period set by default" {
+  cd `chart_dir`
+  local cmd=$(helm template \
+      -s templates/connect-inject-deployment.yaml \
+      --set 'connectInject.enabled=true' \
+      . | tee /dev/stderr |
+      yq '.spec.template.spec.containers[0].command' | tee /dev/stderr)
+
+  local actual=$(echo "$cmd" |
+    yq 'any(contains("-health-checks-reconcile-period"))' | tee /dev/stderr)
+  [ "${actual}" = "true" ]
```

Comment:
could you add an assertion here that `-enable-health-checks-controller=true`? or maybe in a separate test based on the name of this one?

Reply:
the default value for `-enable-health-checks-controller` is `false` so I'm not sure that it would be the right test here. Surely when we do set it enabled by default it would make sense to add this as a test!
Thoughts?

Reply:
My comment I think is more around the fact that there aren't any tests that check that if `connectInject.healthChecks.enabled` is `true`, then there is `-enable-health-checks-controller=true`.

Reply:
Ah, great point, let me add that! thank you